### DISCUSSION TITLE
Downloading data from a folder shared with me

### DISCUSSION THREAD

**Discussion by j-p-courneya** [abraunegg/onedrive#2395] [Q&A] [answered]
Hello, First, thank you for developing this tool. I am not using it yet but was considering it based on my current use case. A collaborator shared a folder containing almost 600 files and i need to transfer them to my remote server. The current workflow i am going through(which is extrememly time consuming) is access the webpage for the sharepoint folder, manually download to my local computer and then scp/sftp to the server. I was wondering if i'd be able to use your software to go from the shared onedrive/sharepoint folder directly to my server. From what i was seeing your software works with ones own onedrive locations.

**Comment by abraunegg** [comment, marked as answer]
@j-p-courneya 
Correct - you should be able to configure the client to access the required SharePoint location, and sync that data to your desired system
